server/database/queries/interests/getInterests.js

```diff
@@ -1,9 +1,6 @@
 const dbConnection = require('../../config/dbConnection');
 
 exports.getInterests = () => {
-  const sql = {
-    text: '',
-    values: [],
-  };
+  const sql = 'SELECT * FROM interests';
```

Comment:
just something to keep in mind, returning all of the records could kill your server if the DB was large enough ;) If you really need to get all of the records, you'd have to implement a paging mechanism to do it safely (but don't bother with that now)...

Reply:
@11janci, so it's better to specify the record that we need it?
Like 
```js
  SELECT id, name FROM interests
```

Reply:
@11janci thanks for the advice, I get what you mean but the interests are not going to be that much even if it's not a fake data

Reply:
@Fatmasiam what I meant is that if the table had 2 million records, this query would select all of them, making the app less responsive, if not killing it completely. But yes as @yosefalnajjarofficial said it should be fine for this table at this scale, just thought I'd point it out as something to keep in mind and watch out for..